gen_mod_headers

```diff
@@ -140,4 +145,41 @@ for f in $(find . -name "Kconfig*"); do
 	cp $f "$target_dir/$f"
 done
 
+echo Fixing up script binaries...
+
+push "$target_dir"
+
+# Make some backups so we can restore sanity after we're done.
+cp scripts/basic/Makefile .backup.basic.Makefile
```

Comment:
You copy scripts in xscripts and then you backup some files. Why exactly? Cause you will have them in xscripts if needed.